### PR TITLE
add combiners

### PR DESCRIPTION
**Pull request by galengorski**
I am having an issue where when I run `tar_make()`, I run into an error: ```Error: callr subprocess failed: Problem with `mutate()` column `color`.
i `color = ...[]`.
x 'from' must be a finite number``` I think this goes back to the log file created by the `tar_combine(summary_state_timeseries_csv)`, when I view my summary_state_timeseries.csv it looks like: 

```
tar_name,filepath,hash
timeseries_png_WI,NA,7d1052ce21757831
timeseries_png_MN,NA,95e1dbf9b67b3d39
timeseries_png_MI,NA,add660a1dd56db92
timeseries_png_IL,NA,849999a8aca5209b
timeseries_png_IN,NA,f15dae290b19f1e0
timeseries_png_IA,NA,5f6333243918d7b8
```

I think there probably shouldn't be NA values there, but I am having trouble tracking where those came from. Any ideas? I'll keep digging into my code too, thanks for the help!

### CODE REVIEW

File: _targets.R [hunk 1]
```diff
@@ -26,20 +29,38 @@ list(
...
     map_sites("3_visualize/out/site_map.png", oldest_active_sites),
     format = "file"
-  )
+  ),
+  tar_target(timeseries_map_html,
+             map_timeseries(oldest_active_sites,
+                            summary_state_timeseries_csv,
+                            "3_visualize/out/timeseries_map.html"))
+
 )
```
Comment (shamshaw) [inline _targets.R:66]: Just another `format = "file"` missing here 🙂

File: _targets.R [hunk 2]
```diff
@@ -26,20 +29,38 @@ list(
...
     values = tibble(state_abb = states)%>%
       mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, get_state_inventory(sites_info = oldest_active_sites, state_abb)),
     tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
     tar_target(tally, tally_site_obs(nwis_data)),
     tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter)),
-    names = state_abb
+    names = state_abb,
+    unlist = FALSE
   ),
```
Comment (shamshaw) [inline _targets.R]: I think I missed this at first also, but you're missing a `format = "file"` here:

`tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter), format = "file")`

That's the issue that's generating those NA values in the file paths which then causes the map target to fail.

**Review by shamshaw — APPROVED** [review]
Just missing a couple of those pesky `format = "file"` options :) 
otherwise you got it and can go ahead and continue

File: _targets.R (hunk 1 above)
Comment (shamshaw) [reply _targets.R:66]: oops, this comment is misplaced. The missing `format = "file" is at the very end where the html file target is

Comment (galengorski) [reply _targets.R:66]: Thanks Scott!

**Pull request merged**